subsys/llext/llext_load.c

```diff
@@ -398,6 +405,7 @@ static int llext_map_sections(struct llext_loader *ldr, struct llext *ext,
 					address -= prepad;
 				}
 				bot_ofs -= prepad;
+				region->sh_info = prepad;
```

Comment:
maybe a single line comment here too, referring to the detailed explanation above